tiledb/sm/storage_manager/context.cc

```diff
@@ -201,6 +203,25 @@ size_t Context::get_io_thread_count(const Config& config) {
       std::max(config_thread_count, io_concurrency_level));
 }
 
+common::Logger::Level Context::get_log_level(const Config& config) {
```

Comment:
This function doesn't need to be a member function of `Context`; it can be a freestanding function.

Reply:
Done.